WordPress/WordPressUITests/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -141,4 +141,14 @@ class BlockEditorScreen: BaseScreen {
     static func isLoaded() -> Bool {
         return XCUIApplication().navigationBars["Gutenberg Editor Navigation Bar"].buttons["Close"].exists
     }
+    @discardableResult
+    func openBlockPicker() -> BlockEditorScreen {
+            addBlockButton.tap()
+        return BlockEditorScreen()
+    }
+    @discardableResult
+    func closeBlockPicker() -> BlockEditorScreen {
+        editorCloseButton.swipeDown()
```

Comment:
This isn't ideal, but editorCloseButton was the only element on the BlockEditorScreen that wasn't located by string and these tests need to be language agnostic. Changing the other elements would have required more refactoring than we have time for in this project.

Reply:
This action was pretty flaky when I ran the test — the swipe didn't always close the picker. What if for now we try tapping on a specific coordinate outside the picker to close it?

Reply:
I used the same element and set the offsets to 0 so hopefully that works more consistently? The previous method worked for me so please let know if this is just a flaky for you.